controllers/openstackbackuprequest_controller.go

```diff
@@ -398,7 +398,7 @@ func (r *OpenStackBackupRequestReconciler) ensureLoadBackup(instance *ospdirecto
 		}
 
 		// Now try to update the status
-		item.Status.CurrentState = ospdirectorv1beta1.MACWaiting
+		item.Status.CurrentState = ospdirectorv1beta1.MACCondTypeConfigured
```

Comment:
Should this be a waiting condition instead?

Reply:
yes

Reply:
fixed